transform/models/marts/fact_commits.sql

```diff
@@ -0,0 +1,7 @@
+select 
+strftime('%m/%d/%Y', event_date) as date,
```

Comment:
It's generally best not to convert dates to strings (and keep them as dates). Also here, do we really need truncate the date at the day level? Maybe we can keep the original date?